scripts/config-build.py

```diff
@@ -50,16 +50,32 @@ def setup_ats(scriptsdir, buildpath):
 
 parser = argparse.ArgumentParser(description="Configure cmake build.")
 
-parser.add_argument("-bp",
-                    "--buildpath",
-                    type=str,
-                    default="",
-                    help="specify path for build directory.  If not specified, will create in current directory.")
-
-parser.add_argument("-ip",
-                    "--installpath", 
-                    type=str, default="",
-                    help="specify path for installation directory.  If not specified, will create in current directory.")
+group = parser.add_mutually_exclusive_group()
+group.add_argument("-bp",
+                   "--buildpath",
+                   type=str,
+                   default="",
+                   help="Specify path for build directory. If both `--buildpath` and `--buildrootdir` are not specified, the build directory will be created in current directory, in a subdirectory named `build-<config-file-name>-<buildtype>`. The `--buildpath` option is not compatible with `--buildrootdir`.")
+
+
+group.add_argument("-br",
+                   "--buildrootdir",
+                   type=str,
+                   default="",
+                   metavar="BUILD_ROOT_DIR",
+                   help="Specify path for root of build directory. The build directory will be created as `BUILD_ROOT_DIR/build-<config-file-name>-<buildtype>`. The `--buildrootdir` option is not compatible with `--buildpath`.")
+
+group = parser.add_mutually_exclusive_group()
+group.add_argument("-ip",
+                   "--installpath",
+                   type=str, default="",
+                   help="specify path for root of installation directory.  If not specified, will create in current directory (in a subdirectory named install-<config-file-name>-<buildtype>). Not compatible with -br --installrootdir, --installpath should be sufficient.")
```

Comment:
```suggestion
                   type=str,
                   default="",
                   help="Specify path for installation directory. If both `--installpath` and `--buildpathdir` are not specified, the install directory will be created in current directory, in a subdirectory named `install-<config-file-name>-<buildtype>`. The `--installpath` option is not compatible with `--installrootdir`.")
```

Reply:
Indeed, I'll fix it.
Sorry for removing the backquotes also, as they didn't have any effect on formattiong I though it was an error.

Reply:
Yes, the back quotes have no effect, other than indicating that we are dealing with some "code".
But more important, it's nice to get the doc homogeneous.